<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_Stack</class>
 <widget class="QWidget" name="Frm_Stack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
color: #c2c2c3;
background: #343639;
}

QGroupBox {
border: 1px solid black;
border-radius: 1px
}

QPushButton {
border: no;
/*outline: no;*/
min-width: 80px;
min-height: 50px;
image: url(&quot;:/Res/image/button_80x50_normal.png&quot;);
}

QPushButton:pressed {
color: black;
border: no;
min-width: 80px;
min-height: 50px;
image: url(&quot;:/Res/image/button_80x50_active.png&quot;);
}

#lbStkPic {
border: 1px solid black;
}
QLineEdit {
 background: qlineargradient(
  x1:0, y1:0, x2:0, y2:1,
  stop:0 gray,
  stop: 0.2 white
  stop:1 white
 );
 border-radius: 1px;
 border: 1px solid black;
 min-height: 30px;
 color: black;
}

QComboBox {
border: 1px solid black;
min-height: 30px;
outline: no;
}

QComboBox::drop-down{
border: 1px solid #2773b4;
min-width: 30px;
}

QComboBox::down-arrow {
image: url(&quot;:/Res/image/downarrow.png&quot;);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
top: 1px;
left: 1px;
}

QComboBox QAbstractItemView {
color: white;
border: 1px solid black;
selection-background-color: #2773b4;
}



</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1" columnstretch="1,1,1">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_18">
       <property name="title">
        <string>启动位置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_65">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>进入点:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="TLineEdit" name="leStartPoint"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_67">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>堆叠速度(%):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="TLineEdit" name="leReadypoint"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_66">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>堆叠起点:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="TLineEdit" name="leStackSpd"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_22">
       <property name="title">
        <string>顺序</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_26" columnstretch="1,2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_41">
          <property name="text">
           <string>第一:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="cbBoxOrderFst3">
          <item>
           <property name="text">
            <string>X方向</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Y方向</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Z方向</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cbBoxOrderFst2">
          <item>
           <property name="text">
            <string>X方向</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Y方向</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Z方向</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_42">
          <property name="text">
           <string>第二:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cbBoxOrderFst1">
          <property name="maxVisibleItems">
           <number>10</number>
          </property>
          <item>
           <property name="text">
            <string>X方向</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Y方向</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Z方向</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_43">
          <property name="text">
           <string>第三:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lbStkPic">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_17">
       <property name="title">
        <string>X轴</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8" columnstretch="1,3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_76">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>方向:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cbBoxOrientX">
          <item>
           <property name="text">
            <string>正向</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>反向</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_77">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>堆叠数量:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="TLineEdit" name="leStkNumX"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_78">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>堆叠间距(mm):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="TLineEdit" name="leStkInterX"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox_20">
       <property name="title">
        <string>Y轴</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_24" columnstretch="1,2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_79">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>方向:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cbBoxOrientY">
          <item>
           <property name="text">
            <string>正向</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>反向</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_80">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>堆叠数量:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="TLineEdit" name="leStkNumY"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_81">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>堆叠间距(mm):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="TLineEdit" name="leStkInterY"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QGroupBox" name="groupBox_21">
       <property name="title">
        <string>Z轴</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_25" columnstretch="1,3">
        <item row="1" column="1">
         <widget class="TLineEdit" name="leStkNumZ"/>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cbBoxOrientZ">
          <item>
           <property name="text">
            <string>正向</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>反向</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_83">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>堆叠数量:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_82">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>方向:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="TLineEdit" name="leStkInterZ"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_84">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>堆叠间距(mm):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="rightMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnStkNew">
       <property name="text">
        <string>新建</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStkOpen">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStkSave">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStkDel">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TLineEdit</class>
   <extends>QLineEdit</extends>
   <header>tlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
